Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/PicassoUtils.java

```diff
@@ -19,6 +19,10 @@
 
 public class PicassoUtils
 {
+
+  // Corner radius for images in dp
+  private static final int CORNER_RADIUS = 8;
```

Comment:
The old corner radius was 4, and the new one is 8, yet the corner radius looks the same to me in the screenshot. This smells like there's a bug somewhere. For instance, did you make sure to treat this as dp and not px?

Reply:
Turns out I misunderstood the drawRoundRect method. You are right, that is in px.

Reply:
Simply specifying the corner radius in px isn't a good way to go. It will cause the corners to be different sizes depending on the pixel density of the display.